packages/core/test/toast/toasterTests.ts

```diff
@@ -121,4 +121,17 @@ describe("Toaster", () => {
             "mutation side effect!",
         );
     });
+
+    describe("with autoFocus set to true", () => {
```

Comment:
_oh_ because it's a prop on the `Toaster` instance. ok well then just fix the redundancy in `it` name. no need to repeat "autoFocus is set to true"